server/src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -577,6 +581,10 @@ public ConnectionManager getConnectionManager() {
         return connectionManager;
     }
 
+    public TransportVersion getMinTransportVersion() {
```

Comment:
I believe this will follow the 'never going backwards' rule, as nodes can't connect to the active cluster in the first place if they make the version go backwards